README.md

```diff
@@ -388,6 +405,18 @@ Executing Logstash diagnostics with a non-default port
 sudo ./diagnostics.sh --host 10.0.0.20 --type logstash-local --port 9607
 ```
 
+Executing Kibana diagnostics in the same server where kibana is running
+ 
+   ```$xslt
+   sudo ./diagnostics.sh --host localhost --port 5601 --type kibana-local
+   ```
+
+Running the kibana-api type to suppress system call and log collection and explicitly configuring an output directory (this is also the option that need bo used when collecting the diagnotic for Kibana in **Elastic cloud**).
+ 
+   ```$xslt
```

Comment:
all the others ES, Logstash in the README has the same format.

Reply:
To be a bit more explicit, here is the text copied from the raw file with your changes (I escapted the \` marks to avoid it terminating the code block). Notice the indentation difference:

```
Executing Logstash diagnostics with a non-default port

\`\`\`$xslt
sudo ./diagnostics.sh --host 10.0.0.20 --type logstash-local --port 9607
\`\`\`

Executing Kibana diagnostics locally from the same server where Kibana is running
 
   \`\`\`$xslt
   sudo ./diagnostics.sh --host localhost --port 5601 --type kibana-local
   \`\`\`

Running the `kibana-api` type to suppress system call and log collection and explicitly configuring an output directory (this is also the option that needs to be used when collecting the diagnostic for Kibana in **Elastic Cloud**).
 
   \`\`\`$xslt
   sudo ./diagnostics.sh --host 2775abprd8230d55d11e5edc86752260dd.us-east-1.aws.found.io --port 9243 --type kibana-api -u elastic --password --ssl -o /home/user1/diag-out
   \`\`\`
 
Executing against a remote host with full collection, using sudo, and enabling trust where there's no known host entry. Note that the diagnostic is not executed via sudo because all the privileged access is on a different host.

\`\`\`$xslt
./diagnostics.sh --host 10.0.0.20 --type remote -u someuser --password --ssl --remoteUser someuser --remotePass --trustRemote --sudo`
\`\`\`
```